recipes/cppfront/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conan import ConanFile
+
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get, rmdir
+
+import os
+
+required_conan_version = ">=1.52.0"
+
+class CppfrontConan(ConanFile):
+    name = "cppfront"
+    description = "Cppfront is a experimental compiler from a potential C++ 'syntax 2' (Cpp2) to today's 'syntax 1' (Cpp1)"
+    topics = ("cppfront", "cpp2")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/hsutter/cppfront"
+    license = "CC-BY-NC-ND-4.0"
+    settings = "os", "arch", "compiler", "build_type"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {"gcc": "11",
+                "Visual Studio": "16.9",
+                "clang": "13",
+                "apple-clang": "13",
+                }
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def export_sources(self):
+        copy(self, "CMakeLists.txt", self.recipe_folder, os.path.join(self.export_sources_folder, "src"))
+
+    def package_id(self):
+        del self.info.settings.compiler
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+
+    def validate(self):
+        if self.info.settings.get_safe("compiler.cppstd"):
```

Comment:
```suggestion
        if self.settings.get_safe("compiler.cppstd"):
```
Let's try without info here, just to see if it passes.